### ISSUE TITLE
--recursive Flags

### ISSUE THREAD

**Issue by meowndor** [glotlabs/gdrive#8]
Hi, does `--recursive` Flags hasn't fully implemented yet?

Given this commands `gdrive files download <FILE_ID>` and replace `<FILE_ID>` with directory ID instead gives me an error: 
```
Error: 'Folder X' is a directory, use --recursive to download directories
```
I've tried with `--recursive` 
```
gdrive files download --recursive <DIR ID>
``` 
it gives me another error:
```
error: Found argument '--recursive' which wasn't expected, or isn't valid in this context

  If you tried to supply '--recursive' as a value rather than a flag, use '-- --recursive'

Usage: gdrive files download [OPTIONS] <FILE_ID>

For more information try '--help'
```

**Comment by prasmussen** [comment]
Yeah, not implemented yet.

**Comment by prasmussen** [comment]
This is now implemented in v3.3.0
